app/ai/voice/agents/breeze_buddy/services/daily/daily.py

```
@@ -58,6 +62,81 @@ async def daily_completion_function(
+    proc = await asyncio.create_subprocess_exec(
+        sys.executable,
+        "-m",
+        "app.ai.voice.agents.breeze_buddy.services.daily.bot_runner",
+        stdin=asyncio.subprocess.PIPE,
+        env=child_env,
+    )
+    assert proc.stdin is not None  # PIPE above guarantees it
+    proc.stdin.write(payload.encode())
+    await proc.stdin.drain()
+    proc.stdin.close()
+    asyncio.create_task(_reap_bot_process(proc, lead_id))
```

Ruff (0.15.21)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)